docs/visualize/tsvb.asciidoc

```diff
@@ -40,6 +40,8 @@ image:images/tsvb-markdown.png["Markdown visualization"]
 
 Table:: Display data from multiple time series by defining the field group to show in the rows, and the columns of data to display.
 
+Only the latest number in a data series will be shown. Under Panel Options > Interval the series length can be defined.
```

Comment:
For the second sentence, would you mind changing it to:

To define the series length, select *Panel options*, then enter the *Interval*.

Reply:
Sure thing